spark/src/main/java/org/opensearch/sql/spark/rest/RestAsyncQueryManagementAction.java

```diff
@@ -194,7 +194,7 @@ public void onResponse(
                           CancelAsyncQueryActionResponse cancelAsyncQueryActionResponse) {
                         restChannel.sendResponse(
                             new BytesRestResponse(
-                                RestStatus.OK,
+                                RestStatus.NO_CONTENT,
```

Comment:
what is the reason for changing the HTTP status here

Reply:
+1

Reply:
https://http.dev/delete
```
A [200 OK](https://http.dev/200) status code indicates that the resource has been removed and it will include a message for the client that describes the status.

A [204 No Content](https://http.dev/204) status code indicates that the resource has been removed but there is no message body to further describe the action or the status.
```
Delete APIs usually send nothing except for 204 status. I need to cross check with the response.